src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -1,18 +1,27 @@
 import { Box, Sans } from "@artsy/palette"
 import SwitchBoard from "lib/NativeModules/SwitchBoard"
 import { Router } from "lib/utils/router"
+import { Schema, track } from "lib/utils/track"
 import React from "react"
 import { Text } from "react-native"
 
 interface ArtworkExtraLinksProps {
   consignableArtistsCount: number
 }
 
+@track()
 export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
   handleTap(href: string) {
     SwitchBoard.presentNavigationViewController(this, href)
   }
 
+  @track(() => {
+    return {
+      action_name: Schema.ActionNames.ConsignWithArtsy,
+      action_type: Schema.ActionTypes.Tap,
+      context_module: Schema.ContextModules.ArtworkExtraLinks,
+    } as any
```

Comment:
Do these `as any`s speak to the need for a type update somewhere?

Reply:
May want to pair on the typing here. I spent some time poking around with the typing here with @l2succes a while ago and then just recently again and it seemed complicated.

Reply:
If it's very complicated I will defer to @ds300 for some 👀!